Assets/Scripts/Characters/EnemyRunner.cs

```diff
@@ -12,33 +12,43 @@ public class EnemyRunner : Enemy{
     private float speed = 2F;
     private int attackDmg = 30;
     private int searchAngle = 180;
+	private float spawnTimer = 3.5F;
+	private float spawnRoarDelay = 2F;
     float parryTime = 5F;
 
+	public AudioClip runnerRoarClip;
+
+	bool spawning = false;
     bool attacking = false;
     bool parryable = false;
 
     Animator anim;
 
 	enum runnerState{
+		spawning,
 		idle,
 		follow,
 		attack,
 		dead
 	};
 
-	private runnerState currentState = runnerState.idle;
+	private runnerState currentState;
```

Comment:
you can initialize the state to spawning here. It's better to keep the initialization close to the declaration, rather than in start. Use start for things which need to be created in the engine before finding them, but statics and so on can be used immediately.